File: Step-1/CPP/Step-1.1/3. If Else Statements/ifElse.cpp
#include <bits/stdc++.h>
using namespace std;

string compareIfElse(int a, int b) {
	// Write your code here
	string ans = " ";
	if(a > b){
		ans = "greater";
	}
	else if(a < b){
		ans = "smaller";
	}
	else{
		ans = "equal";
	}
	return ans;
}

int main(){
    int a,b; cin>>a>>b;
    string ans = compareIfElse(a,b);
    cout << ans << endl;
    return 0;
}

File: Step-1/Python/Step-1.1/3. If Else Statements/ifElse.py
def compare_if_else(a, b):
    # Write your code here
    if a > b:
        return "greater"
    elif a < b:
        return "smaller"
    else:
        return "equal"

def main():
    a, b = map(int, input().split())
    ans = compare_if_else(a, b)
    print(ans)

if __name__ == "__main__":
    main()